.github/workflows/clang-tools-amd64.yml

```diff
@@ -2,7 +2,7 @@ name: clang-tools-static-amd64
 
 on:
   push:
-    branches: [ master, macos-arm-squash ]
+    branches: [ master, separate-artifact-uploads ]
```

Comment:
For the maintainer of the project:

Consider allowing to run CI on pull requests, as most, if not all, the possible contributions to the repo are for CI, there's not much more.
But, you could select to have first time contributors have their CI jobs be approved before running, or if you feel active enough, for everyone not in the organization/rights to the repo.

You then keep building for pushes on master branch, and the rest goes on pull requests.

It is not a suggestion for this particular PR, but seeing the churn just to see if a branch works makes me think of it.

Reply:
nice, yea makes sense